### DISCUSSION TITLE
'Include' and 'Select' information in DMMF

### DISCUSSION THREAD

**Discussion by chrishoermann** [prisma/prisma#16729] [Q&A] [answered]
I'm currently building a generator for prisma and I'm wondering where in the DMMF the information for generating the `Select`, `Include` and all corresponding types are stored?

I cant find them in the `DMMF.OutputType` nor in the `DMMF.InputType`. Should there be a representation in one of this types or are they generated based on the datamodel?

I'm grateful for any hint.

**Comment by chrishoermann** [comment, marked as answer]
I found all the necessary inforamtion in `DMMF.Document.Schema.outputObjectTypes.model`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
